packages/tenant-process/test/tenant.integration.test.ts

```diff
@@ -228,7 +228,7 @@ describe("Integration tests", () => {
 
         expect(writtenPayload.tenant).toEqual(toTenantV1(expectedTenant));
       });
-      it("Should throw operation forbidden", async () => {
+      it("Should throw operation forbidden for insufficient permissions", async () => {
```

Comment:
still not sure which permissions we are checking here.
In our endpoints we can have permissions based on role, organization ownership, producer vs consumer, etc.
It is useful to indicate which permission we are validating because the use case is comprehensible right from the description (and because in general we can have tests for each permission level)